src/pages/en/reference/api-reference.md

```diff
@@ -158,6 +158,29 @@ You could pass a callback function that renders our the message:
 <div>Hello world!</div>
 ```
 
+### `Astro.self`
+
+`Astro.self` allows Astro components to be recursively called. This behaviour lets you call the same Astro component over again by defining `<Astro.self>` within the component template.  Allowing you to easily iterate over large data stores and nested data-structures.
```

Comment:
```suggestion
within the component template, allowing you to easily iterate over large data stores and nested data-structures.
```
I love the last bit, but it's not a sentence on its own. It's either going to be after a comma fromt whatever you decide for before, or it would need to be something like:

"This allows you to easily iterate..."